root/static/scripts/edit/externalLinks.js

```diff
@@ -278,7 +278,7 @@ export class ExternalLinksEditor
                 }
                 isOnlyLink={this.state.links.length === 1}
                 key={link.relationship}
-                removeCallback={_.bind(this.removeLink, this, index)}
+                handleRemoveCallback={_.bind(this.removeLink, this, index)}
```

Comment:
This looks like a wrong use of `handle`, since it's a prop name and not a component method. "handleRemoveCallback" also doesn't make a lot of sense. :) `onRemove` should do fine.